mlperf_logging/result_summarizer/result_summarizer.py

```diff
@@ -183,8 +184,19 @@ def _is_organization_folder(folder):
         return False
     return True
 
+def _csv_writer(path, lines_list):
+    with open(path, 'w', newline='') as csvfile:
+        res_writer = csv.writer(csvfile, delimiter=',',quotechar='"', quoting=csv.QUOTE_MINIMAL)
+        res_writer.writerows(lines_list)
 
-def summarize_results(folder, ruleset):
+class _csv_(object):
```

Comment:
Rename `_csv_` to something like `_CsvBuilder` or `_CSVBuilder`.

Also, `_csv_` in its current form serves no purpose other than just being a list, so why creating a class for it at all?

Reply:
removed in its entirely